### ISSUE TITLE
A problem in ShuffleNet

### ISSUE THREAD

**Issue by rwenjie957** [kuangliu/pytorch-cifar#163]
When I run shufflenet in PyCharm,  an error occured.
![problem](https://github.com/kuangliu/pytorch-cifar/assets/120168616/dacd5282-ef23-4052-bc08-88485f6fcdbe)
I find the problem is in line 27.
![problem1](https://github.com/kuangliu/pytorch-cifar/assets/120168616/af0eb132-421a-4921-b1bc-bd5ef08237e3)
Perhapse it is due to the problem of python versions.
The environment of my python is 3.10, the result of dividing two integers is a floating point number, but nn.Conv2d needs an integer.
So we just need to convert it into an integer.
![poblem3](https://github.com/kuangliu/pytorch-cifar/assets/120168616/d152db81-3f21-41ab-97b0-80f346914180)
